mlxtend/evaluate/tests/test_mcnemar_test.py

```diff
@@ -38,20 +38,20 @@ def test_corrected_false():
 def test_exact():
     tb = np.array([[101, 121], [59, 33]])
 
-    chi2, p = (None, 4.4344492637555101e-06)
+    p = 4.4344492637555101e-06
     chi2p, pp = mcnemar(tb, exact=True)
 
-    assert chi2 is None
+    assert chi2p is None
     assert_almost_equal(p, pp, decimal=7)
     assert p < 4.45e-06
```

Comment:
Question: Why `assert p < 4.45e-06` is required?

It should be `assert pp < 4.45e-06`? (but I think `assert_almost_equal(p, pp, decimal=7)` verify it)

Reply:
Honestly, this has been many years ago and I don't recall. Looking at it now, I think it is redundant can can be removed. Maybe it was a temporary check I forgot to remove